lib/discourse_api.rb

```diff
@@ -39,9 +39,17 @@ def get_badge_id
   end
 
   def create_topic
-    entry = Entry.includes(:user).where(users: {username: @username}).first
     url = "#{DISCOURSE_URL}/posts?api_key=#{API_KEY}&api_username=#{@username}"
-    post = "##{entry.name}\n  Repository url: #{entry.url}\n  About the project:\n  #{entry.about}\n  "
+    user = User.includes(:entry, team: :entry).find_by_username(@username)
+    if user.entry
+      entry = user.entry
+      post = "##{entry.name}\n  Repository url: #{entry.url}\n  About the project:\n  #{entry.about}\n  "
+    elsif user.team.entry
+      entry = user.team.entry
+      user_list = user.team.users.collect{|u| "@#{u.username}"}.join('/n  ')
```

Comment:
Prefer `map` over `collect`.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.